README.md

```diff
@@ -89,7 +96,9 @@ git submodule update --init
 The service uses PostgreSQL database for data storage.
 
 * Create and activate virtual environment
-* Run: `pip install -r requirements.txt`
+* Prod mode, run: `pip install -r requirements.txt`
```

Comment:
in the 2.18 release, I've switched to Poetry so this stuff will need to be merge-deconflicted

Reply:
Will do!